Cargo.toml

```diff
@@ -13,6 +13,7 @@ version = "0.1.4"
 
 [features]
 default = ["jpegxl-sys/default", "with-image"]
+build-jpegxl = ["jpegxl-sys/build-jpegxl"]
```

Comment:
Ideally the build should be automatic 🤔

Reply:
It's a workaround about `docs.rs`. They don't allow network connection so you can't pull anything during the build. 😑